test/testData.js

```diff
@@ -325,6 +325,11 @@ var testPoints = [
     ll: [2, 1],
     xy: [1057002.4052152266, 110955.14117382761]
   },
+  {
+    code: '+proj=utm +lon_0=-3 +ellps=GRS80 +lat_1=0.5 +lat_2=2 +n=0.5',

```

Comment:
this zone 30, but I have to put in the lon_0 that corresponds to zone 29, which seems wrong unless I'm missing something